coreos-base/oem-ec2-compat/files/grub-ec2.cfg

```diff
@@ -5,3 +5,10 @@ set oem_id="ec2"
 # Blacklist the Xen framebuffer module so it doesn't get loaded at boot
 # Disable `ens3` style names, so eth0 is used for both ixgbevf or xen.
 set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0"
+
+if [ "$grub_platform" = pc ]; then
+	set linux_console="console=ttyS0,115200n8"
```

Comment:
Very intentional, if the config has customized the linux console it is responsible for configuring grub too.

Oh, but you can actually drop console, so grub also doesn't do the dual output, only writing to serial_com0